src/main/java/lotto/domain/WinningTier.java

```diff
@@ -0,0 +1,65 @@
+package lotto.domain;
+
+import lotto.dto.WinningResponseDto;
+import lotto.dto.WinningTierResponseDto;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class WinningTier {
+
+    private static final int WINNING_TIER_START_INDEX = 0;
+    private static final int DEFAULT_VALUE = 0;
+    private static final int START_TIER_VALUE = 1;
+    private static final int PLUS_UNIT = 1;
+    private static final int END_TIER_VALUE = 5;
+    private static final int WINNING_TIER_DEFAULT_VALUE = 0;
+
+    private HashMap<Integer, Integer> winningTier;
+
+    public WinningTier() {
+        this.winningTier = new HashMap<>();
+        initializeWinningTier();
+    }
+
+    private void initializeWinningTier() {
+        for (int tier = START_TIER_VALUE; tier <= END_TIER_VALUE; tier++) {
+            this.winningTier.put(tier, WINNING_TIER_DEFAULT_VALUE);
+        }
+    }
+
+    public void estimate(List<Long> correctWinningsCount, List<Boolean> correctBonuses) {
+        HashMap<Integer, Integer> tempWinningTier = new HashMap<>();
+        for (int i = WINNING_TIER_START_INDEX; i < correctWinningsCount.size(); i++) {
+            Long correctWinningCount = correctWinningsCount.get(i);
+            boolean correctBonus = correctBonuses.get(i);
+
+            WinningStatistics confirmResult = WinningStatistics.confirm(correctWinningCount, correctBonus);
+            int rank = confirmResult.getRank();
+            if (winningTier.containsKey(rank)) {
+                tempWinningTier.put(rank, tempWinningTier.getOrDefault(rank, DEFAULT_VALUE) + PLUS_UNIT);
+            }
+        }
+        tempWinningTier.forEach((rank, count) -> winningTier.merge(rank, count, Integer::sum));
+    }
```

Comment:
이함수는 15줄을 넘지 않는다 라는 규칙에 빡빡해 보이는 점이 있네요 !!
```suggestion
    WinningStatistics confirmResult = WinningStatistics.confirm(correctWinningCount, correctBonus);
            int rank = confirmResult.getRank();
            if (winningTier.containsKey(rank)) {
                tempWinningTier.put(rank, tempWinningTier.getOrDefault(rank, DEFAULT_VALUE) + PLUS_UNIT);
            }}
```

위의 함수를 tempWinningTier를 인자로 받아 개수를 세도록 분리할 수 있을것 같아요!